### PR TITLE
feat: add column span full for form fields

### PR DESCRIPTION
**Pull request by aymanalhattami**
column span full

### CODE REVIEW

**Comment by atmonshi** [comment]
Aren't fields fully spaned by default and controllable by the section columns?
Or am I missing something? :)

### Default
Screenshot 2023-08-27 at 5 52 07 PM

### Sections Columns:
Screenshot 2023-08-27 at 5 52 25 PM

**Comment by aymanalhattami** [comment]
> Aren't fields fully spanned by default and controllable by the section columns?
 
exactly

but there is another case:
I need some fields to span the full width and other fields to be divided as each row has 2 fields "If we assume that the sections has two columns"


👇"more details" field span the full width while the others fields are in 3 columns 
bolt


👇column span full for 'more details' field is set to true, while column span full of other fields are set to false and they appear in 3 columns based on section columns
more details

**Comment by atmonshi** [comment]
got you 👍 thanks

File: src/Concerns/Schemata.php [hunk 1]
```diff
@@ -300,7 +300,7 @@ public static function getSectionsSchema(): array
                     'default' => 1,
                     'md' => 2,
                     'xl' => 3,
-                    '2xl' => 4,
+                    '2xl' => 3,
```
Comment (atmonshi) [inline src/Concerns/Schemata.php:303]: why change this? I think with `2x1`  there is enough space for 4 column

Comment (aymanalhattami) [reply src/Concerns/Schemata.php:303]: I was trying to set 3 columns for 2xl and I forgot to set it back to 4.

Comment (atmonshi) [reply src/Concerns/Schemata.php:303]: I will fix it NP

**Pull request merged**